submission/server/main.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Submission proxy server runs (pre-)certs multi-Log submission complying with
```

Comment:
I think you need to pick a binary name and use it here (`submission_server` maybe)?

Reply:
Updated